helm/cluster-azure/templates/_helpers.tpl

```diff
@@ -154,6 +154,17 @@ List of admission plugins to enable based on apiVersion
 - /opt/bin/calculate_kubelet_reservations.sh
 {{- end -}}
 
+{{- define "kubeadm.controlPlane.privateNetwork.preCommands" -}}
```

Comment:
can we have a comment for all the `/etc/hosts` mutation stuff why this is needed + a link to an upstream issue (if one exists) (IIRC this has to be done on flatcar only)